generator/index.js

```diff
@@ -22,7 +22,7 @@ module.exports = (api, options) => {
     })
 
     api.render('./templates/style')
-    api.injectImports(api.entryFile, `import '@/assets/scss/thisvui.scss'`)
+    api.injectImports(api.entryFile, `import 'thisvui/dist/sass/thisvui.scss'`)
```

Comment:
Hey @mu545 thank you so much, i am kind new in writing plugins for vue-cli, do you recommend handling source styles inside the template folder?

Reply:
Handling source style inside template folder will make the source style can be customized by users.
Import all main of sources style be need in '@/assets/scss/thisvui.scss' so it allows a user to customize.

Reply:
I see, thanks for the help and insight, i'll create a issue to handle this.